wakeword_training.ipynb

```
@@ -677,161 +515,113 @@
+    "    precision = precision_score(all_labels, all_preds, average='binary', zero_division=0)\n",
+    "    recall = recall_score(all_labels, all_preds, average='binary', zero_division=0)\n",
+    "    f1 = f1_score(all_labels, all_preds, average='binary', zero_division=0)\n",
+    "\n",
+    "    print(f\"\\n📊 Validation Set Performance:\")\n",
     "    print(f\"   Accuracy: {accuracy:.4f}\")\n",
     "    print(f\"   Precision: {precision:.4f}\")\n",
     "    print(f\"   Recall: {recall:.4f}\")\n",
     "    print(f\"   F1-Score: {f1:.4f}\")\n",
-    "    \n",
+    "\n",
     "    # Confusion matrix\n",
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)